Source/Core/DiscIO/NANDImporter.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Common/FileUtil.h"
+
+namespace DiscIO
+{
+class NANDImporter
+{
+public:
+  static void ImportNANDBin(const std::string& path_to_bin);
```

Comment:
Is there any reason you cannot just construct an instance of this class when you need to import a NAND image, instead of having only static methods?